doc/contributing.rst

```diff
@@ -21,6 +21,37 @@ of the users who use the package.
 page as the maintainers about changes or enhancements before too much
 coding is done saves everyone time and effort!
 
+Installing developer version
+----------------------------
+
+The standard installation procedure can be found on
+:ref:`install_python_and_mne_python` but if you want to install a
+developer version directly from the source code in order to contribute,
+you can follow those steps:
+
+  .. code-block:: console
+
+     $ git clone git@github.com:mne-tools/mne-python.git
+     $ cd mne-python
+     $ conda env create -f environment.yml
+     $ conda activate mne
+     $ pip install -e .
+
+As optional steps, you can also add:
+
+  .. code-block:: console
+
+     $ conda install ipython sphinx
```

Comment:
https://github.com/mne-tools/mne-python/blob/master/environment.yml#L19 suggests ipython is already installed by the environment.yml

Reply:
I will remove it from install then